liesl/files/labrecorder/cli_wrapper.py

```diff
@@ -19,14 +20,14 @@ def find_lrcmd_os(platform: str) -> Path:
     root = Path(
         pkg_resources.resource_filename("liesl", "files/labrecorder/lib")
     )
-    if "linux" in platform:
-        path_to_cmd = root / "LabRecorderCLI"
-    elif "win" in platform:
+    if platform.startswith('win'):
```

Comment:
"darwin" is linux so needed to change this to make it work

Reply:
Desn't sys.platform return 'darwin' on MacOs? https://docs.python.org/3/library/sys.html#sys.platform

Reply:
indeed !

Reply:
I use the `startswith` idiom now to prevent Mac being confused with Linux as "win" is in the identifier for Mac